modules/outbound-command-event-handler/src/domain/bulk_transaction_agg/index.ts

```diff
@@ -159,6 +184,135 @@ export class BulkTransactionAgg extends BaseAggregate<BulkTransactionEntity, Bul
             .setIndividualTransfer(this._rootEntity.id, entity.id, entity.exportState());
     }
 
+    async getBulkQuotesTotalCount() {
+        const repo = this._entity_state_repo as IBulkTransactionEntityRepo;
+        return repo.getBulkQuotesTotalCount(this._rootEntity.id);
+    }
+
+    async setBulkQuotesTotalCount(totalCount: number) : Promise<void> {
+        await (<IBulkTransactionEntityRepo> this._entity_state_repo)
+            .setBulkQuotesTotalCount(this._rootEntity.id, totalCount);
+    }
+
+    async getBulkQuotesSuccessCount() {
+        const repo = this._entity_state_repo as IBulkTransactionEntityRepo;
+        return repo.getBulkQuotesSuccessCount(this._rootEntity.id);
+    }
+    
+    async setBulkQuotesSuccessCount(count: number) : Promise<void> {
+        await (<IBulkTransactionEntityRepo> this._entity_state_repo)
+            .setBulkQuotesSuccessCount(this._rootEntity.id, count);
+    }
+
+    async incrementBulkQuotesSuccessCount() : Promise<void> {
+        await (<IBulkTransactionEntityRepo> this._entity_state_repo)
+            .incrementBulkQuotesSuccessCount(this._rootEntity.id);
+    }
+
+    async getBulkQuotesFailedCount() {
+        const repo = this._entity_state_repo as IBulkTransactionEntityRepo;
+        return repo.getBulkQuotesFailedCount(this._rootEntity.id);
+    }
+
+    async setBulkQuotesFailedCount(count: number) : Promise<void> {
+        await (<IBulkTransactionEntityRepo> this._entity_state_repo)
+            .setBulkQuotesFailedCount(this._rootEntity.id, count);
+    }
+
+    async incrementBulkQuotesFailedCount() : Promise<void> {
+        await (<IBulkTransactionEntityRepo> this._entity_state_repo)
+            .incrementBulkQuotesFailedCount(this._rootEntity.id);
+    }
+
+    async addBulkBatchEntity(entity: BulkBatchEntity) : Promise<void> {
+        await (<IBulkTransactionEntityRepo> this._entity_state_repo)
+            .setBulkBatch(this._rootEntity.id, entity.id, entity.exportState());
+    }
+
+    async setBulkBatchById(id: string, bulkBatch: BulkBatchEntity): Promise<void> {
+        await (<IBulkTransactionEntityRepo> this._entity_state_repo)
+            .setBulkBatch(this._rootEntity.id, id, bulkBatch.exportState());
+    }
+
+    async createBatches(maxItemsPerBatch: number) : Promise<void> {
```

Comment:
Please put a comment here as to what this method does.

Reply:
Done